.pylintrc

```diff
@@ -6,13 +6,22 @@
 init-import=yes
 
 [MESSAGES CONTROL]
-disable=C0116, C0330, RST203, RST301
+disable=
+    C0116,  # missing-function-docstring
+    C0330,  # for black formatting
+    RST203,
+    RST301,
 
 [MASTER]
 ignore=conf.py
 
+[TYPECHECK]
+ignored-modules=
+    amplitf.interface,
+
 [DESIGN]
 # Minimum number of public methods for a class (see R0903). Default is 2.
 # We decrease this to 1, since some interface classes just have a single
 # (public) method
+max-args=7  # default: 5
```

Comment:
can you shift the comment down

Reply:
Which comment? `# default`?

Reply:
The comment above corresponds to the line below.

Reply:
Oops yes

Reply:
7bb4b205900ab26f37c27e0ad11ee0eef0cd4ac5